src/test/ui/borrowck/issue-52967-edition-2018-needs-two-phase-borrows.rs

```diff
@@ -1,22 +1,22 @@
 // This is a regression test for #52967, where we discovered that in
 // the initial deployment of NLL for the 2018 edition, I forgot to
 // turn on two-phase-borrows in addition to `-Z borrowck=migrate`.
+// The original version of this test used matches, which no longer
+// use 2 phase-borrows in MIR borrowck.
```

Comment:
Hmm. This is a *regression* test; the idea is to ensure that the original code continues to compile and run with the same semantics.

My inclination is to just leave the test alone (or delete it entirely, if one is concerned about the number of tests we have been accumulating; but I don't don't think we should do such deletion in an ad hoc manner).

If you want to add a separate test that deliberately tests two-phase borrows in the manner shown here, that would be fine with me.